tests/parser/features/test_reverting.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from eth_tester.exceptions import TransactionFailed
+
+from vyper.utils import keccak256
+
+pytestmark = pytest.mark.usefixtures("memory_mocker")
+
+
+def method_id(method_str: str) -> bytes:
+    return keccak256(bytes(method_str, "utf-8"))[:4]
```

Comment:
this feels like it should go in vyper/utils.py. i'm ok with it as it is as long as there is a note like `# TODO should move to utils.py`

Reply:
moving it there makes sense imo, but there's already a function in there called `abi_method_id`

the difference between that one and this one is that `abi_method_id` returns an int instead of bytes. this probably works in other places in the compiler, but it breaks something in the usage here.

when `method_id` returns bytes, we can concatenate them to other bytes like we do in the test:

`revert_bytes = method_id("NoFives(uint256)") + (5).to_bytes(32, "big")`

with a returned integer, this doesn't work

I think a function called `method_id`/`abi_method_id` should return bytes, and we should rename `abi_method_id` to something like `method_id_int`. happy to take suggestions on naming convention here, I'm not the best at coming up with names